javascript/javascript3/week1/WeatherApp/main.js

```diff
@@ -0,0 +1,130 @@
+const btn1 = document.getElementById('btnGetCity');
```

Comment:
Really really nice job! Really good use of functions to implement the API call function and the render functions - there's really not a lot I would change here. I just have some nitty gritty comments here, but it's just semantics really. 🚀